spec/spec_helper.rb

```diff
@@ -20,3 +20,9 @@
     end
   end
 end
+
+# This loads the module data, needed to test different OSes.
+#RSpec.shared_context "hiera", :shared_context => :metadata do
+#  let (:hiera_config) { 'hiera.yaml' }
+#  Hiera.new({:config => 'hiera.yaml'})
+#end
```

Comment:
can you remove that change. it would anyway be overwritten by modulesync.

Reply:
I don't believe it is needed for module level data ? If it is there is a 

```
<%- if @configs['hiera_config'] -%>

RSpec.configure do |c|
  c.hiera_config = <%= @configs['hiera_config'] %> 
end
<%- end -%>
```
in the template for `spec_helper.rb`

Reply:
I had added that becuase I missed the fact that I didn't name the keys in Debian.yaml properly, which I fixed in https://github.com/voxpupuli/puppet-nftables/pull/66/commits/f8c4097f31bc18500a1c7e05dcd203a3e3b5207b

I will remove the commented code.